building/build-debs/homeworld-admin-tools/src/access.py

```diff
@@ -67,10 +67,8 @@ def access_ssh(add_to_agent=False):
     if add_to_agent:
         # TODO: clear old identities
         if subprocess.call(["ssh-add", "--", keypath]) != 0:
-            print("ssh-add returned non-zero exit code. do you have a ssh-agent?")
-            print("or, perhaps your agent is broken; consider killing it and launching a new one.")
-            command.fail("*** ssh-add failed! ***")
-
+            fail_hint = "ssh-add returned non-zero exit code. do you have a ssh-agent?\nor, perhaps your agent is broken; consider killing it and launching a new one."
```

Comment:
Might want to have this be multiple lines for readability, either with `"""..."""` notation or by having a second line with `fail_hint +=`.